etc/config.sample.toml

```diff
@@ -71,10 +71,12 @@
 
   # CacheMaxMemorySize is the maximum size a shard's cache can
   # reach before it starts rejecting writes.
+  # Values without a size suffix (e.g. 30k, 20m, 10g) are in bytes.
```

Comment:
Can you update this to say: `Valid size suffixes are k, m, or g (case insensitive). Values without a size suffix are in bytes.`